docs/xostor/xostor.md

```diff
@@ -323,6 +323,78 @@ Only XCP-ng 8.2.1 is currently supported and with a maximum of 7 machines per po
 
 See this documentation: [RPU](/management/updates/#rolling-pool-update-rpu).
 
+## Upgrade
+
+If you are reading this documentation, we assume that you want to upgrade a pool on which XOSTOR is deployed, i.e. change the version of XCP-ng, for example from 8.2 to 8.3.
+For updates that don't change the version number of XCP-ng (bugfixes, security fixes), see [the update section](#update).
+
+### 1. Prerequisites
+
+- All hosts must be up to date on the version of XCP-ng you are currently using. For this refer to [the update section](#update).
+- HA must be disabled on your pool.
+- Ensure all nodes are reachable and resources are in "OK" state via XO's XOSTOR view. Alternatively, you can use the CLI:
+```
+linstor n l
+linstor r l
+linstor adv r
+linstor sp l
+```
+
+### 2. Upgrade steps
+
+From this point we can proceed to upgrade your XOSTOR-enabled pool.
+
+An upgrade can take quite a long time so we recommend disabling the auto-evict mechanism during this procedure to avoid bad behavior.
+On the host where the controller is running:
+```
+linstor controller set-property DrbdOptions/AutoEvictAllowEviction False
+```
+
+For each host of the pool (starting with the master), follow the instructions given in [this guide](../installation/upgrade/#-upgrade-via-installation-iso-recommended). Just one important thing: don't try to upgrade using CLI or network ISO installer. Always use the latest full ISO that can be found [here](https://xcp-ng.org/download/).
+
+:::warning
+If you have this error during upgrade, you must download the latest full ISO version as documented above:
+```
+Cannot upgrade host with LINSTOR using a package source that does not have LINSTOR.  Please use as package source the repository on the dedicated ISO.
+```
+:::
+
+If you want to make sure that everything is going well so as not to impact your production, we recommend these manual checks after each host reboot:
+
+- Check if there is an issue with the resources:
+```
+linstor r list
+linstor advise r # Give possible fix commands in case of problems.
+```
+
+- Check in XOA that the PBD of the SR of this host is connected. If not, do so.
```

Comment:
Might add to check `linstor node list` to see if the node is correctly connected after coming back up?

Reply:
Or is the PBD being connected enough?